examples/benchmark/scene_test_1.py

```diff
@@ -143,6 +144,7 @@ def on_mouse_move(self, event):
             self.update()
 
     def on_mouse_wheel(self, event):
+        prof = Profiler()  # noqa
```

Comment:
how about we make this a decorator? That way this could just be:
```
@vispy_profile
on_mouse_wheel(self, event):
```
WDYT? That way anything we want to be profiled we can just add a decorator to.